src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -80,7 +81,10 @@ protected static ConfigDef baseConfigDef() {
           .define(RETRY_BACKOFF_MS_CONFIG, Type.LONG, 100L, Importance.LOW,
                   "How long to wait in milliseconds before attempting to retry a failed indexing request. "
                   + "This avoids retrying in a tight loop under failure scenarios.",
-                  group, ++order, Width.SHORT, "Retry Backoff (ms)");
+                  group, ++order, Width.SHORT, "Retry Backoff (ms)")
+          .define(IGNORE_MAPPING_ERRORS_CONFIG, Type.BOOLEAN, false, Importance.LOW,
+          "Whether to ignore mapper_parsing_exceptions thrown by Elasticsearch upon indexing malformed documents.",
+                  group, ++order, Width.SHORT, "Ignore Mapping Errors mode");
```

Comment:
Do we really want the documentation to include the `mapper_parsing_exceptions` term, or would it be better to remove the underscores?

Also, every time we update or change any of the configuration files, we'd like the PR to also update the `docs/configuration_options.rst` contents. The easiest way to do that is to run the [`main(...)` method in the `ElasticsearchSinkConnectorConfig`](https://github.com/stitchfix/kafka-connect-elasticsearch/blob/master/src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java#L295-L297) class, and they carefully copy/paste the contents accordingly.

I do this through my IDE or on the command line with:

    mvn exec:java -Dexec.mainClass="io.confluent.connect.elasticsearch/ElasticsearchSinkConnectorConfig" -Dexec.classpathScope="test"

Reply:
Actually, after thinking about this a bit more, using boolean here is simple but it offers no room for handling other behaviors. Have you thought about using an enumeration to define the different behaviors, similar to the `BehaviorOnNullValues` enum used in #165? The advantage is that we could define other behaviors besides "ignore", such as "fail" and "write" (as suggested by @nabeelz6).

Reply:
Thanks for the review @rhauch. I like the idea of being able to configure different options for what to do when a conflict happens. I'll take a look at #165 for inspiration and update the PR. Will also update the `config_options.rst`

Reply:
Thanks @zzbennett!

Reply:
FYI I went on vacation for a week and am recovering from the flu now, but I am still planning on implementing the suggestion! Shooting for middle of next week.